test/parser/hive/syntax/delete.test.ts

```diff
@@ -0,0 +1,16 @@
+import HiveSQL from '../../../../src/parser/hive';
+import { readSQL } from '../../../helper';
+
+const parser = new HiveSQL();
+
+const features = {
+    deletes: readSQL(__dirname, 'delete.sql'),
+};
+
+describe('Hive Delete Syntax Tests', () => {
```

Comment:
```suggestion
describe('HiveSQL Delete Syntax Tests', () => {
```
单测描述和其他SQL类型保持一致
![image](https://github.com/DTStack/dt-sql-parser/assets/22880945/e2f50ceb-c916-4a10-ad36-93ad7028e08c)